blockly-ws-search/workspace_search.js

```diff
@@ -64,6 +64,20 @@ class WorkspaceSearch {
      * @type {boolean}
      */
     this.searchOnInput = true;
+
+    /**
+     * HTML container for the workspace search bar.
+     * @type {?Element}
```

Comment:
since html is also in the name of this, I think type should change to HTMLElement